cluster/Vagrantfile

```diff
@@ -86,15 +98,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         # Download rhel7.2 box from https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software
         # Add it as rhel7 vagrant box add rhel-cdk-kubernetes-7.2-29.x86_64.vagrant-virtualbox.box --name=rhel7
         c.vm.box = 'rhel7'
+        config.ssh.insert_key = false
+      elsif ENV['CONTIV_NODE_OS'] && ENV['CONTIV_NODE_OS'] == 'ubuntu'
+        c.vm.box = 'ubuntu/xenial64'
```

Comment:
This string appears five times in this file... I'd consider making it a constant